skins/cat17/package.json

```diff
@@ -123,5 +127,17 @@
   },
   "browserify-shim": {
     "jquery": "global:jQuery"
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "ts",
+      "json",
+      "vue"
+    ],
+    "transform": {
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.tsx?$": "ts-jest"
+    }
```

Comment:
Hmm why wouldn't we need them? This is telling jest to transpile `.vue` and `.ts` to `js` as far as I understand.
I followed the instructions from the [Vue docs.](https://vue-test-utils.vuejs.org/guides/using-with-typescript.html#setting-up-jest)

Reply:
Because we're not using jest in the "main" cat17 scripts yet, only in the sub-folder/ecosystem of `address-form`. The jest dependency is both in `skins/cat17/package.json` and `skins/cat17/address-form/package.json` and it's only needed in `skins/cat17/address-form/package.json`

You can still add the `test:unit` command to `skins/cat17/package.json` and relay it to `npm --prefix=address-form run test:unit`

Reply:
Oh I totally did not see this is `cat17`'s `package.json`. :see_no_evil: 
Initially I installed Jest for `cat17` but then moved it to `address-form`.